src/ci/docker/run.sh

```diff
@@ -7,6 +7,7 @@ export MSYS_NO_PATHCONV=1
 script=`cd $(dirname $0) && pwd`/`basename $0`
 
 image=""
+
 dev=0
 
 while [[ $# -gt 0 ]]
```

Comment:
?

Reply:
Going to mess around with the CI so gonna use this PR to trigger builds. Let me know if there is a better way (this does feel hacky)

Reply:
Makes sense. I will close this PR in that case!